### PR TITLE
[DE] Update `osu!supporter`

### PR DESCRIPTION
**Pull request by respektive**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!supporter/de.md [hunk 1]
```diff
@@ -1,67 +1,55 @@
...
 
-![Der Vergleich: Inaktiver und aktiver osu!supporter.](img/status.jpg?1 "Der Vergleich: Inaktiver und aktiver osu!supporter.")
+*Tipp: Dieser Abschnitt wird nur angezeigt, nachdem du osu!supporter erhalten hast.
 
-Sobald die Transaktion abgeschlossen ist, wird auf deinem Profil unter "Recent Activity" folgendes stehen: "{Name des Benutzers} has become an osu! supporter - thanks for your generosity!" (sofern das der erste Tag auf dem jeweiligen Account ist). Sofern schonmal osu!Supporter geholt wurde, wird stattdessen "{Name des Benutzers} has once again chosen to donate to the osu! cause - thanks for your generosity!" stehen. Wenn du dir den für dich selber gekauft hast, wirst du unter deinem Profilbild das osu!supporter Abzeichen finden. Um nachzuprüfen, wann dieser abläuft, [kehre zur Seite zurück, woher du diesen gekauft hast](https://osu.ppy.sh/home/support).
+Um deinen verbleibenden osu!supporter-Tag-Status und die Gesamtsumme der Beiträge und Tag-Käufe zu überprüfen, gehe auf die Seite [Das Spiel unterstützen](https://osu.ppy.sh/home/support) und scrolle nach unten, um einen ähnlichen Bereich wie oben abgebildet zu sehen.
```
Comment (The-Last-Cookie) [inline wiki/osu!supporter/de.md:55]:
```suggestion
Um den Status deines verbleibenden osu!supporter-Tags und die Gesamtsumme der Beiträge und Tag-Käufe zu überprüfen, gehe auf die Seite [Das Spiel unterstützen](https://osu.ppy.sh/home/support) und scrolle nach unten, um einen ähnlichen Bereich wie oben abgebildet zu sehen.
```
Removes one of the two hyphens, to improve readability.

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
Looks good to me otherwise :+1:

File: wiki/osu!supporter/de.md (hunk 1 above)
Comment (respektive) [reply wiki/osu!supporter/de.md:55]: Definitely reads better!

File: wiki/osu!supporter/de.md [hunk 2]
```diff
@@ -1,67 +1,55 @@
...
+## osu!supporter erhalten
+
+![osu!store Produktbanner](img/store-product.jpg?1 "osu!supporter Produktbanner aus dem osu!store")
+
+Um osu!supporter zu erhalten, melde dich an (oder [registriere dich](/wiki/Registration)) und gehe dann auf die [Shop-Seite](https://osu.ppy.sh/store/products/supporter-tag). Verwende den Schieberegler oder die Monatsauswahl (unterhalb des Schiebereglers), um die Dauer des osu!supporter-Tags auszuwählen. Bei einer anderen Dauer ändert sich der Preis, den du zahlen musst. Die Währung ist der amerikanische Dollar (USD). Wenn dein Land eine andere Währung verwendet, benutze bitte einen Währungsrechner wie [OANDA](https://www.oanda.com/currency/converter/) oder [Google](https://www.google.com/search?q=usd+wechselkurs), um den Preis zu überprüfen. Die auf der osu!store-Seite angezeigten Preise gelten für jeden einzelnen Account.
+
+### Auswählen der Dauer
 
-## Einleitung
+!{Kauf von osu!supporter](img/selecting-duration.jpg?1 "Auswahl des Benutzers und der Dauer für osu!supporter")
```
Comment (Walavouchey) [inline wiki/osu!supporter/de.md:19]:
```suggestion
![Kauf von osu!supporter](img/selecting-duration.jpg?1 "Auswahl des Benutzers und der Dauer für osu!supporter")
```

File: wiki/osu!supporter/de.md [hunk 3]
```diff
@@ -1,67 +1,55 @@
...
+Das osu!supporter Abzeichen ändert sich, je nachdem, wie lange du das Spiel unterstützt hast. Wenn du das Spiel weniger als 1 Jahr lang unterstützt, erhältst du 1 Herz; für 1–5 Jahre erhältst du 2 Herzen; und für mehr als 5 Jahre erhältst du 3 Herzen.
 
-Nun hast du verschiedene Bezahloptionen, die weltweit verwendet werden. Empfehlenswerte Bezahlmethoden sind [PayPal](https://www.paypal.com) oder Kreditkarten. Zwar gibt es auch andere Möglichkeiten wie etwa per SMS Bezahlung, allerdings wird da eine zusätzliche Gebühr angerechnet und ist daher nicht unbedingt die kostengünstigste Bezahlmethode. Wenn du weniger als deine Gesamtkosten ("Total Cost") in "dorrah" bezahlst (zusätzliche Kosten, sofern nicht Paypal benutzt wird, sind inkludiert), erhälst du nicht deine(n) Abzeichen.
+### Support-Status abfragen
 
-### Dauer
+![osu!supporter-Tag mit drei Monaten Restlaufzeit](img/status.jpg?1 "Drei Monate Restlaufzeit des osu!supporter-Tags")
 
-![Der Vergleich: Inaktiver und aktiver osu!supporter.](img/status.jpg?1 "Der Vergleich: Inaktiver und aktiver osu!supporter.")
+*Tipp: Dieser Abschnitt wird nur angezeigt, nachdem du osu!supporter erhalten hast.
```
Comment (Walavouchey) [inline wiki/osu!supporter/de.md:53]:
```suggestion
*Tipp: Dieser Abschnitt wird nur angezeigt, nachdem du osu!supporter erhalten hast.*
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/osu!supporter/de.md (hunk 2 above)
Comment (respektive) [reply wiki/osu!supporter/de.md]: blame german keyboard for dumb bracket palcements :smile:

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**